manifest.schema.json

```diff
@@ -0,0 +1,11 @@
+{
+    "type": "array",
+    "items": {
+      "type": "object",
+      "properties": {
+        "metadata": { "type": "string" },
+        "snippet": { "type": "string" }
+      },
+      "required": ["metadata", "snippet"]
```

Comment:
[q] I believe product said that the snippet is optional for viewType `functionSnippetWithNumberedGuide`.

Reply:
well, I officially do not understand the "guide" thing, then, if it doesn't require a snippet.

Reply:
I know this is for the input manifest.json. Just wanted to callout that the output on s3 is expecting the following format from the SPEC.
```
{
"snippetExamples": [
  {
    “metadata”: <relative path to the metadata file for the snippet>,
    "snippet": <relative path to the snippet file>
  }
],
"version": <git commit hash or git tag>
}

```

Reply:
I'll post the snippet being optional question in the project channel. I remember it being mentioned and I took notes on it. I do think it would make this more messy to add validations